### ISSUE TITLE
quotes seem to be stripped

### ISSUE THREAD

**Issue by Fuseteam** [fredldotme/git-confined-snap#6]
it seems that quoted arguments aren't passed through properly

```
fuseteam@tuxecure:~/.local/src/fuseteam.github.io$ GIT_TRACE=1 git commit -m "hello world"
13:39:34.775086 git.c:472               trace: built-in: git commit -m hello world
```

the correct trace should `git commit -m "hello world"`

**Comment by fredldotme** [comment]
This would have to be issued as a bugreport on snapd's launchpad.net page. I can add some fuel to get eye balls there during the Summit.

**Comment by Fuseteam** [comment]
i can try that, not as familiar with launchpad yet

**Comment by Fuseteam** [comment]
i think this should do it: https://bugs.launchpad.net/snapd/+bug/2085371

**Comment by fredldotme** [comment]
Feel free to point them to the snapcraft.yaml.

**Comment by fredldotme** [comment]
Also please test upcoming edge in around half an hour, I might have spotted something.

**Comment by Fuseteam** [comment]
i think it works :D

I'll verify tommorow

**Comment by fredldotme** [comment]
Fixed by a2c51f31d696e22e35a66532a771744253a23d15 and released to stable.

**Comment by Fuseteam** [comment]
thanks for the quick fix :D
